src/main/java/org/sonarlint/intellij/core/SonarQubeEventNotifications.java

```diff
@@ -27,6 +27,7 @@
 import com.intellij.util.messages.MessageBusConnection;
 import java.time.ZonedDateTime;
 import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)